src/Worker.Extensions.DurableTask/Worker.Extensions.DurableTask.csproj

```diff
@@ -32,15 +32,15 @@
     <VersionPrefix>1.1.2</VersionPrefix>
     <VersionSuffix></VersionSuffix>
     <AssemblyVersion>$(VersionPrefix).0</AssemblyVersion>
-    <!-- FileVersionRevision is expected to be set by the CI. -->
+    <!-- FileVersionRevision is expected to be set by the CI.  -->
     <FileVersion Condition="'$(FileVersionRevision)' != ''">$(VersionPrefix).$(FileVersionRevision)</FileVersion>
   </PropertyGroup>
   
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Core" Version="1.16.0" />
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Abstractions" Version="1.3.0" />
-    <PackageReference Include="Microsoft.DurableTask.Client.Grpc" Version="1.1.1" />
-    <PackageReference Include="Microsoft.DurableTask.Worker.Grpc" Version="1.1.1" />
+    <PackageReference Include="Microsoft.DurableTask.Client.Grpc" Version="1.2.2" />
+    <PackageReference Include="Microsoft.DurableTask.Worker.Grpc" Version="1.2.2" />
```

Comment:
Should it be 1.1.2 instead? It seems that there is no 1.2.* released .

Reply:
`1.2.0` was released here: https://github.com/microsoft/durabletask-dotnet/releases/tag/v1.2.0 :) .
So at first I wanted to release `1.2.1`, but a buggy version of 1.2.1 was pushed to ADO and since we cannot replace ADO versions, we increase the version again and go straight to 1.2.2 instead

Reply:
Oh my bad. I didn't notice there is recent release. Please ignore my above comment I will approve the PR.